project/ChatBuild.scala

```diff
@@ -23,7 +23,8 @@ object ChatBuild extends Build {
         testFrameworks += new TestFramework("utest.runner.Framework"),
         libraryDependencies ++= Seq(
           "org.scala-js" %%% "scalajs-dom" % "0.8.0",
-          "com.lihaoyi" %%% "utest" % "0.3.0" % "test"
+          "com.lihaoyi" %%% "utest" % "0.3.0" % "test",
+          "com.lihaoyi" %%% "upickle" % "0.2.8"
```

Comment:
Maybe switch those lines around so that the test dependencies are at the end. Also below.